src/Console/Application.php

```diff
@@ -119,7 +119,7 @@ public function getLongVersion(): string
         $commit = '@git-commit@';
         $versionCommit = '';
 
-        if ('@'.'git-commit@' !== $commit) { /** @phpstan-ignore-line as `$commit` is replaced during phar building */
+        if ('@git-commit@' !== $commit) { /** @phpstan-ignore-line as `$commit` is replaced during phar building */
```

Comment:
blocker, please revert.
we also have stan-ignore for it.
basically it must concat on execution time.

Phar builder is replacing `@git-commit@` string with actual string value. We want phar builder to replace on line 119 but not 122

Reply:
reverted